spec/controllers/games_controller_spec.rb

```diff
@@ -74,4 +74,28 @@
       expect(response).to have_http_status(:not_found)
     end
   end
+
+  describe 'games#forfeit action' do
+    it 'should show white player as winner if black player forfeits' do
+      white = FactoryBot.create(:user)
+      black = FactoryBot.create(:user)
+      sign_in white
```

Comment:
Did you mean `sign_in black`?

Reply:
No sign in white because whoever creates the game becomes the white player.